index.html

```diff
@@ -200,7 +200,22 @@ <h1>About me</h1>
   <div class="container-fluid bg-primary contact-section">
     
   </div>
-  
+
+  <form class="container-fluid">
+    <div class="form-div row">
```

Comment:
good work, 
but i think need more work on formating the form,
remove class form-div may fix some issues.